pkg/utils/k8sutil/pravega.go

```diff
@@ -17,21 +17,29 @@ import (
 )
 
 func ConfigMapNameForBookie(clusterName string) string {
-	return fmt.Sprintf("%s-bookie-config", clusterName)
```

Comment:
Please try not to bundle unrelated work.

Reply:
You are right, I added this because it's a small naming change and I was just trying be consistent in the way we name resources. It doesn't make sense to add suffixes like `-config`, `-service`, `-deployment` because they are already differentiated by their resource type.